resources/assets/uad_lists.json

```
@@ -20510,6 +20528,24 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.oplus.olc",
+    "list": "Oem",
+    "description": "Olc\nSystem logger.\n\nPithus analysis: https://beta.pithus.org/report/805177a4193d101e281cc31987694c85adea75d5737e43b08bfa668a6eb71c95",
```

```suggestion
    "description": "Olc\nSystem logger.\n\nPithus analysis: https://beta.pithus.org/report/805177a4193d101e281cc31987694c85adea75d5737e43b08bfa668a6eb71c95",
```
Maybe OLC stands for and is related to https://en.wikipedia.org/wiki/Open_Location_Code? What do you think? Because the Pithus analysis also says it has location/GPS access.

---

It might be, but I'm confused. It seems to have access to a quite a few permissions and maybe it happens to include location/GPS access as well? So, I can't say for sure if it's related to [Open Location Code](https://en.wikipedia.org/wiki/Open_Location_Code).

---

I've thought the same thing, because when we look at permissions, it doesn't explicitly request the location/GPS, but Pithus also says it's able to get the location. Maybe through 4G/5G? and not location/GPS? Maybe we should mention that we're not sure whether it's for location purposes or not?

---

I suppose we can. Not sure what it would use it for other than collecting telemetry

---

Can you push a commit regarding this?

---

[Added](https://github.com/0x192/universal-android-debloater/pull/711/commits/f74ac6d09630b015b340fc6d052ceb3a8b2f7f7c). Should it be changed from `Recommended`? Doesn't seem to break any system functionality or affect battery life.

---

I think recommended is fine, if it doesn't break anything.